database/migrations/2020_09_13_215758_create_tasks_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTasksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->integer('search_id')->unsigned();
+            $table->string('Sector');
+            $table->string('Status')->default("to_do");
+            $table->integer('Group');
+            $table->dateTime('Start');
+            $table->dateTime('End');
+            $table->string('Type');
+            $table->longText('Description');
+            $table->integer('Gpx')->default(0);
+            $table->string('GpxFileName')->nullable();
+            $table->binary('GpxFile')->nullable();
```

Comment:
Same comment here to put the first letter in lowercase. Also, all strings should include the nullable() option